content/docs/cdn-links.md

```diff
@@ -1,37 +1,37 @@
 ---
 id: cdn-links
-title: CDN Links
+title: Tautan CDN
 permalink: docs/cdn-links.html
 prev: create-a-new-react-app.html
 next: hello-world.html
 ---
 
-Both React and ReactDOM are available over a CDN.
+React dan ReactDOM tersedia melalui CDN.
 
 ```html
 <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
 <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
 ```
 
-The versions above are only meant for development, and are not suitable for production. Minified and optimized production versions of React are available at:
+Versi di atas hanya dimaksudkan untuk pengembangan dan tidak cocok untuk produksi. Versi produksi React yang telah diperkecil dan dioptimalkan tersedia di:
 
 ```html
 <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
 <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
 ```
 
-To load a specific version of `react` and `react-dom`, replace `16` with the version number.
+Untuk memuat versi spesifik dari `react` dan `react-dom`, ganti `16` dengan nomor versi.
 
-### Why the `crossorigin` Attribute? {#why-the-crossorigin-attribute}
+### Mengapa Atribut `crossorigin`? {#why-the-crossorigin-attribute}
 
-If you serve React from a CDN, we recommend to keep the [`crossorigin`](https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes) attribute set:
+Jika Anda menjalankan React dari CDN, kami merekomendasikan untuk tetap membiarkan set atribut [`crossorigin`](https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes):
 
 ```html
 <script crossorigin src="..."></script>
 ```
 
-We also recommend to verify that the CDN you are using sets the `Access-Control-Allow-Origin: *` HTTP header:
+Kami juga merekomendasikan untuk memverifikasi bahwa CDN yang Anda gunakan menetapkan HTTP *header* `Access-Control-Allow-Origin: *`:
 
 ![Access-Control-Allow-Origin: *](../images/docs/cdn-cors-header.png)
 
-This enables a better [error handling experience](/blog/2017/07/26/error-handling-in-react-16.html) in React 16 and later.
+Hal ini memungkinkan [pengalaman penanganan kesalahan](/blog/2017/07/26/error-handling-in-react-16.html) di React 16 dan yang lebih baru.
```

Comment:
```suggestion
Hal ini dapat membuat [pengalaman penanganan kesalahan](/blog/2017/07/26/error-handling-in-react-16.html) yang lebih baik di React versi 16 dan versi yang lebih baru.
```

Kalau begini bagaimana mas?, soalnya menurutku kalo yang lama kontex dari kalimat nya agak hilang di bagian  _This enables a better_

Reply:
that is indeed a lot more readable. will update!